vendor.conf

```diff
@@ -0,0 +1,57 @@
+# go-runc client for runc; following master HEAD on 12/16/2016
+github.com/crosbymichael/go-runc 2fdd3cd06f7443a55f6eb14ead2cac507506e718
+# distribution code from Docker; v2.6.0-rc.1 tag
+github.com/docker/distribution v2.6.0-rc.1
+# go-metrics client to prometheus; following master HEAD on 12/16/2016
+github.com/docker/go-metrics 0f35294225552d968a13f9c5bc71a3fa44b2eb87
+# prometheus client; v0.8.0
+github.com/prometheus/client_golang v0.8.0
+# prometheus client model; following master HEAD on 12/16/2016
+github.com/prometheus/client_model fa8ad6fec33561be4280a8f0514318c79d7f6cb6
+# prometheus common library; following master HEAD on 12/16/2016
+github.com/prometheus/common 195bde7883f7c39ea62b0d92ab7359b5327065cb
+# prometheus procfs; following master HEAD 12/16/2016
+github.com/prometheus/procfs fcdb11ccb4389efb1b210b7ffb623ab71c5fdd60
+# beorn7/perks; following master HEAD 12/16/2016
+github.com/beorn7/perks 4c0e84591b9aa9e6dcfdf3e020114cd81f89d5f9
+# matttproud/golang_protobuf_extensions; v1.0.0 release
+github.com/matttproud/golang_protobuf_extensions v1.0.0
+# go-units from Docker; latest release = 0.3.1
```

Comment:
nit: I think the notation of  "latest" can quickly become outdated. Also, mentioning the version tag in the comment seems too verbose.

So, how about:
```
# go-units from Docker; latest release at 12/16/2016
```